config/routes.rb

```diff
@@ -34,4 +34,6 @@
   get "/admin/completed-orders", to: "admin/orders#index_completed"
 
   resources :charges
+
+  post 'twilio/connect_customer' => 'twilio#connect_customer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.